Schema Statements.sql

```diff
@@ -0,0 +1,67 @@
+CREATE DATABASE  IF NOT EXISTS `products`;
+USE `products`;
+
+DROP TABLE IF EXISTS `category`;
+CREATE TABLE `category` (
+  `id` int NOT NULL,
+  `name` varchar(45) NOT NULL,
+  PRIMARY KEY (`id`)
+);
+
+
+INSERT INTO `category` VALUES
+  (1,'foods'),
+  (2,'drinks'),
+  (3,'entertainment'),
+  (4,'pets'),
+  (5,'baby'),
+  (6,'houseware'),
+  (7,'health'),
+  (8,'other');
+
+DROP TABLE IF EXISTS `unit`;
+CREATE TABLE `unit` (
+  `id` int NOT NULL,
+  `name` varchar(45) NOT NULL,
+  PRIMARY KEY (`id`)
+);
+
+INSERT INTO `unit` VALUES 
+  (0,'g'),
+  (1,'ml'),
+  (2,'each');
+
+
+DROP TABLE IF EXISTS `origin`;
+CREATE TABLE `origin` (
+  `id` int NOT NULL AUTO_INCREMENT,
```

Comment:
I see you are using AUTO_INCREMENT here, but it is missing in some of the other tables. Could we please have AUTO_INCREMENT consistently?